config/runtime/boot/fastboot.jinja2

```diff
@@ -10,6 +10,11 @@
         format: cpio.newc
         overlays:
           lava: true
+          modules:
+            url: '{{ node.artifacts.modules }}'
+            compression: xz
+            format: tar
+            path: /
```

Comment:
These may not be necessary here, after reviewing other similar code that already exists.

That said, I am not 100% sure.

Reply:
@EmbeddedAndroid, thank you for double-checking! Appreciated. 👍 

I wasn't sure if the extra attributes were necessary after checking how other templates handle modules. However, none of the other ones include the `overlays` section under `ramdisk`, indeed.